simtools/simtel/simtel_config_writer.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python3
+
+import logging
+
+
+__all__ = ['SimtelConfigWriter']
+
+
+class SimtelConfigWriter:
+    '''
+    SimtelConfigWriter writes sim_telarray configuration files. \
+    It is designed to be used by model classes (TelescopeModel and ArrayModel) only.
+
+    Methods
+    -------
+    writeTelescopeConfigFile(configFilePath, parameters)
+        Writes the sim_telarray config file for a single telescope.
+    writeArrayConfigFile(configFilePath, layout, telescopeModel, siteParameters)
+        Writes the sim_telarray config file for an array of telescopes.
+    writeSingleMirrorListFile(
+        mirrorNumber,
+        mirrors,
+        singleMirrorListFile,
+        setFocalLengthToZero=False
+    )
+        Writes the sim_telarray mirror list file for a single mirror.
+    '''
+    TAB = ' ' * 3
```

Comment:
What does this do? Three spaces?

Reply:
This is the indentation that has to be used in sim_telarray config files. It's used when writing sim_telarray config files.